src/logging.c

```diff
@@ -379,6 +390,7 @@ static const token_map_entry_t tokens[] = {
   {.token = "function_name", .handler = expand_function_name},
   {.token = "file_name", .handler = expand_file_name},
   {.token = "time", .handler = expand_time_as_seconds},
+  {.token = "date_time_with_ms", .handler = expand_time_as_date},
```

Comment:
So this is called `date_time_with_ms`, but as far as I can tell below, we are putting "nanoseconds" into the last bit.  Which is it?  Milliseconds or nanoseconds?

Reply:
We truncate the ns to ms using `".%.3s"` this is much more readable and for my use case I don't need ns precision only ms